docs/src/rules/class-methods-use-this.md

```diff
@@ -6,55 +6,91 @@ further_reading:
+- The method's outcome is related to the object on which it's invoked, including possibly its state.
 
-It's possible to have a class method which doesn't use `this`, such as:
+	```js
+	const array1 = [1, 2, 3];
+	const array2 = [4, 5, 6];
 
-```js
-class A {
-    constructor() {
-        this.a = "hi";
-    }
+    // Using the `includes()` method on different objects gives different results:
+	array1.includes(1); // true
+	array2.includes(1); // false
 
-    print() {
-        console.log(this.a);
-    }
+	// Modifying the state of an object may change the outcome of its instance methods:
+	array2.push(1);
+	array2.includes(1); // true
+	```
+
+- The method doesn't make sense to be used without an associated object. (For example, it doesn't make sense to call `Array#includes()` without an array to operate on.)
 
+If a class instance method does not use `this`, that normally means that it does not access any instance state and therefore doesn't need to be a method.
+Therefore, it can *sometimes* be refactored into an [ordinary function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions) or a [static method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static), which may better communicate intent to users of the API.
+
+It's possible to have a class method which doesn't use `this`, such as:
+
+```js
+class Person {
     sayHi() {
-        console.log("hi");
+        console.log("Hi!");
     }
 }
 
-let a = new A();
-a.sayHi(); // => "hi"
+const person = new Person();
+person.sayHi(); // => "Hi!"
 ```
 
-In the example above, the `sayHi` method doesn't use `this`, so we can make it a static method:
+In the example above, the `sayHi` method doesn't use `this`, so we can make it an ordinary function:
 
 ```js
-class A {
-    constructor() {
-        this.a = "hi";
-    }
+// Ordinary function
+function sayHi() {
+	console.log("Hi!");
+}
 
-    print() {
-        console.log(this.a);
-    }
+// No need for `Person` class or any instance thereof
+sayHi(); // => "Hi!"
 
+// Alternately, a static method may be used if it offers a more natural API
+class Person {
     static sayHi() {
-        console.log("hi");
+        console.log("Hi!");
     }
 }
 
-A.sayHi(); // => "hi"
+Person.sayHi(); // => "Hi!"
+
+// Keep in mind that, either way, the following now throws an error,
+// since sayHi() is no longer an instance method!
+//
+// const person = new Person();
+// person.sayHi();
 ```
 
-Also note in the above examples that if you switch a method to a static method, *instances* of the class that call the static method (`let a = new A(); a.sayHi();`) have to be updated to being a static call (`A.sayHi();`) instead of having the instance of the *class* call the method.
+Of course, it's also possible the author simply forgot to use some piece of instance data that they intended to include.
```

Comment:
[Approachability] We don't want to imply this is obvious ("of course") or a dumb/silly mistake ("simply"):

```suggestion
It's also possible the author forgot to use some piece of instance data that they intended to include.
```